src/prettier/index.spec.ts

```diff
@@ -43,6 +36,50 @@ describe('@co-it/schematics:prettier', () => {
         expect.objectContaining({ name: 'node-package' })
       );
     });
+
+    it.each([['prettier'], ['tslint-config-prettier']])(
+      'should install $packageId',
```

Comment:
$packageId is not substituted in this case. Use %p as a placeholder. https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout

Reply:
That's correct.
I already know.
This PR is in Draft.

No Review needed yet. :)